rero_ils/modules/patron_transaction_events/sample_data/sample-scenario-overdue1.json

```diff
@@ -0,0 +1,100 @@
+[
+  {
+    "$schema": "https://ils.rero.ch/schema/patron_transaction_events/patron_transaction_event-v0.0.1.json",
+    "pid": "1",
```

Comment:
instead of `pid=1` move to `pid=ptre1`

Reply:
According to other resources in `data` folder, the pid are always just a digit.
'ptre1' should be used into fixtures